os/drivers/wireless/realtek/wifi_util_interface/rtk_drv.c

```diff
@@ -53,7 +53,7 @@
 		vddbg(RTKDRV_TAG "%s:%d\n", __FILE__, __LINE__); \
 	} while (0)
 
-
+struct netdev *g_rtk_netdev;
```

Comment:
How about adding `static` if it is used in this file only?

Reply:
I'll modify it later.